### PR TITLE
Documentation not Rendering with multi-OpenApi files 

### PR DESCRIPTION
**Pull request by Apoorva64**
### Description
- Implementing solution of replacing invalid with a character here that character is "-"

### Related issue(s)
#1067

- [x] test: tested manually, works.

### CODE REVIEW

File: webapp/src/test/java/io/github/microcks/service/ServiceServiceTest.java [hunk 1]
```diff
@@ -220,6 +220,7 @@ public void testImportServiceDefinitionFromGitLabURL2() {
       List<Resource> resources = resourceRepository.findByServiceId(importedSvc.getId());
       assertEquals(resourceNames.size(), resources.size());
       for (Resource resource : resources) {
+        System.out.println(resource.getName());
```
Comment (lbroudoux) [inline webapp/src/test/java/io/github/microcks/service/ServiceServiceTest.java:223]: Can you remove this one please?

File: commons/model/src/main/java/io/github/microcks/util/Sanitizer.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
+package io.github.microcks.util;
```
Comment (lbroudoux) [inline commons/model/src/main/java/io/github/microcks/util/Sanitizer.java:1]: Can you add the standard copyright header and check your indentation?
I know, I know... we still have this topic on code formatter 😉

Comment (Apoorva64) [reply commons/model/src/main/java/io/github/microcks/util/Sanitizer.java:1]: @lbroudoux The indentation is 3 spaces right?

Comment (Apoorva64) [reply commons/model/src/main/java/io/github/microcks/util/Sanitizer.java:16]: We need to change the .editorconfig for the formatting

indent_size = 2 --> indent_size = 3

Comment (lbroudoux) [reply commons/model/src/main/java/io/github/microcks/util/Sanitizer.java:16]: Yes, I use the `.editorconfig` only for VS Studio when editing Typescript and HTML where indentation is 2 chars. We need to make things clear here ;-)

**Comment by Apoorva64** [comment]
@lbroudoux Is there somthing missing is this PR?

**Comment by lbroudoux** [comment]
Nope! I think we're good to go!

**Review by lbroudoux — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**